[2025-08-05T23:32:11.351+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Ejercicio1.Updatefactsales manual__2025-08-05T23:31:56.558776+00:00 [queued]>
[2025-08-05T23:32:09.757+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Ejercicio1.Updatefactsales manual__2025-08-05T23:31:56.558776+00:00 [queued]>
[2025-08-05T23:32:09.775+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-08-05T23:32:09.939+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): Updatefactsales> on 2025-08-05 23:31:56.558776+00:00
[2025-08-05T23:32:09.988+0000] {standard_task_runner.py:60} INFO - Started process 3448 to run task
[2025-08-05T23:32:10.011+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Ejercicio1', 'Updatefactsales', 'manual__2025-08-05T23:31:56.558776+00:00', '--job-id', '245', '--raw', '--subdir', 'DAGS_FOLDER/ejercicio1.py', '--cfg-path', '/tmp/tmp8c9f5am6']
[2025-08-05T23:32:10.027+0000] {standard_task_runner.py:88} INFO - Job 245: Subtask Updatefactsales
[2025-08-05T23:32:10.412+0000] {task_command.py:423} INFO - Running <TaskInstance: Ejercicio1.Updatefactsales manual__2025-08-05T23:31:56.558776+00:00 [running]> on host 019b67754667
[2025-08-05T23:32:11.599+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Ejercicio1' AIRFLOW_CTX_TASK_ID='Updatefactsales' AIRFLOW_CTX_EXECUTION_DATE='2025-08-05T23:31:56.558776+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-05T23:31:56.558776+00:00'
[2025-08-05T23:32:12.192+0000] {_client.py:1038} INFO - HTTP Request: GET https://qwpzsjaaxuijmqodvslf.supabase.co/rest/v1/StgLogs?select=last_id&table_name=eq.factsales "HTTP/2 200 OK"
[2025-08-05T23:32:12.310+0000] {logging_mixin.py:188} INFO - 
 ---------------NO HAY DATOS NUEVOS PARA INSERTAR EN SUPABASE-----------------
[2025-08-05T23:32:12.313+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-08-05T23:32:12.345+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=Ejercicio1, task_id=Updatefactsales, execution_date=20250805T233156, start_date=20250805T233211, end_date=20250805T233212
[2025-08-05T23:32:12.532+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-05T23:32:12.588+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
